lib/controller.js

```diff
@@ -12,14 +12,13 @@ var semver = require('semver');
 
 // Internal
 var CrashReporter = require('./crash-reporter');
-var discover = require('./discover');
 var init = require('./init');
 var installer = require('./installer');
 var log = require('./log');
 var Menu = require('./menu');
 var updates = require('./update-fetch');
-var provision = require('./tessel/provision');
 var Tessel = require('./tessel/tessel');
+var provision = require('./tessel/provision');
```

Comment:
Reordered this so that certain constants, i.e. `Tessel.LOCAL_AUTH_PATH`, can be added correctly.